src/power_divergence.jl

```diff
@@ -79,6 +81,9 @@ function StatsBase.confint(x::PowerDivergenceTest, alpha::Float64=0.05;
         i = StatsBase.confint(x, alpha*2,method=method, GC=GC)
         Tuple{Float64,Float64}[(i[j][1], 1.0) for j in 1:m]
     elseif tail == :both
+        if method == :auto
+            method = minimum(x.expected) > 100 ? :quesenberry_hurst : :sison_glaz
+        end
         if method == :gold
```

Comment:
`elseif`?

Reply:
No it is not `elseif` - I update the `method` variable value if needed and reuse it later in the other `if`.